lib/archive.rb

```diff
@@ -25,8 +25,7 @@ def self.get_files(archive_path)
         header_position: i,
         mac_bs_file: pathname.include?("__MACOSX") ||
```

Comment:
Not related to your changes but  "__MACOSX" is a folder not a file. Therefore, it still appears in the speedgrader.

![image](https://user-images.githubusercontent.com/31053044/115421712-2ddf3280-a1ca-11eb-90da-6b2860ed642d.png)

Reply:
Just to make sure, does it mean we don't want to display any folders in the archive that contains "__MACOSX"?

Reply:
Yup. That's a folder that mac creates when someone archives something.

Reply:
Cool I'll submit another PR to fix this soon!